### PR TITLE
Operations - Convert Section to Docupipe

### PR DESCRIPTION
**Pull request by tommarkosi**

### CODE REVIEW

**Comment by tommarkosi** [comment]
Looks good via local DocFx test:
image

**Comment by tommarkosi** [comment]
here is what the new numbering looks like rendered:
image

File: content/api-reference/operations/toc.yml [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-  href: operations-resource-usage-data.md
-- name: Billing cycles
-  href: operations-billing-cycles.md
-- name: Health
-  href: operations-health.md
-- name: Metrics
-  href: operations-metrics.md
+- name: 1. Health
+  href: operations-health-query.md
+- name: 2. Logs
```
Comment (GiselleLeeOSI) [inline content/api-reference/operations/toc.yml:3]: You don't need to number them in the toc

Comment (tommarkosi) [reply content/api-reference/operations/toc.yml:3]: I'll remove those

**Review by GiselleLeeOSI — APPROVED** [review]

**Comment by tommarkosi** [comment]
FYI, I don't have access to merge this PR:
image

**Comment by GiselleLeeOSI** [comment]
> FYI, I don't have access to merge this PR:
> image

If you're okay, I'll go ahead and merge. @tommarkosi, let me know!

**Comment by tommarkosi** [comment]
I am okay with these changes, please merge so osisoft:DocuPipe-operations is ready for the DocuPipe merges to come

**Pull request merged**